PluginsAndFeatures/azure-toolkit-for-intellij/resources/hdinsight/templates/scala/scala_local_run/LogQuery.scala

```diff
@@ -17,7 +17,7 @@
  */
 
 
-//package org.apache.spark.examples
+package sample
```

Comment:
How about other samples?

Reply:
Just verified, other samples doesn't hit the issue.